src/pages/docs/infrastructure/accounts/azure/index.md

```diff
@@ -76,25 +97,39 @@ Next, if you want to get even more granular you can constrain the service princi
 
 The reason behind this has to do with the way Octopus queries for the web app resources in Azure. In order to handle scenarios where [ASEs](/docs/deployments/azure/ase/#resource_groups) are being used, Octopus first queries the resource groups and then queries for the web apps within each resource group. When the service principal is assigned **Contributor** on a resource group it seems to implicitly get **Reader** on the subscription, but this doesn't seem to be the case when **Contributor** is assigned directly to a web app, so you have to assign **Reader** explicitly.
 
-### Create an Service Principal with PowerShell {#create-service-principal-account-with-powershell}
+### Create an Azure App Registration via script {#create-app-registration-via-script}
 
-This step shows you how to create a Service Principal with the PowerShell script below, if you would rather use the Azure Portal to create the Service Principal, see [Create an Azure Service Principal With the Azure Portal](#create-service-principal-account-in-azure).
+This step shows you how to script the creation of an Azure Active Directory App Registration
 
 :::div{.hint}
 During the script, you will be prompted to authenticate with Azure. The authenticated user must have administrator permissions in the Active Directory in which the Service Principal is being created.
 :::
 
+<details data-group="infrastructure-accounts-azure">
+<summary>Az CLI</summary>
+
+```bash
+# this script will create a new Azure AD App Registration
+
+subscription='' # Replace with the name or id of your subscription
+appName='' # Replace with your app registration name
+
+az login
+az account set --subscription $subscription
+az ad app create --display-name "$appName" -o table --query "{Id:id,Name:displayName,ClientId:appId}"
+az account show  --query "{Name:name,SubscriptionId:id,TenantId:tenantId}" -o table
+```
+</details>
+
 <details data-group="infrastructure-accounts-azure">
 <summary>Az PowerShell</summary>
 
 ```powershell
-# This script will create a new service principal for you to use in Octopus Deploy using the Az PowerShell modules.  This will work with both PowerShell and PowerShell Core.
+# this script will create a new Azure AD App Registration
 
 $AzureTenantId = "2a681dca-3230-4e01-abcb-b1fd225c0982" # Replace with your Tenant Id
```

Comment:
Nit: Is there a reason this has the tenant specified? This is a pre-existing part of the docs so I'm happy for it to be left as is, the inconsistency is just a bit odd.

Reply:
This is the Azure Active Directory tenant , not an Octopus one

Reply:
🤦‍♂️